Core/Util/PPGeDraw.cpp

```diff
@@ -1222,15 +1221,17 @@ bool PPGeImage::Load() {
 		return false;
 	}
 
-	u32 texSize = width_ * height_ * 4;
+	u32 dataSize = width_ * height_ * 4;
+	u32 texSize = dataSize + height_ * 4;
```

Comment:
Hm, the idea is to add an extra row, right? Shouldn't this be `+ width_ * 4` ?

Reply:
Erm, yes, oops.  Thanks.

-[Unknown]